stories/panel.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text } from '@storybook/addon-knobs';
+
+import Panel from '../src/components/Panel';
+
+const panelContent = <div>Simple panel text</div>;
+const panel = storiesOf('Components/Panel', module)
+  .add('default', () =>
+    <Panel
+      headerText={text('headerText', 'Default Panel Title')}
+      body={panelContent}
+    />
+  )
+  .add('all panels with styles', () => {
+    return [
+      <Panel
+        headerText="Default Panel"
+        style={text('style', '')}
```

Comment:
The `style` prop is optional; might as well leave it out to demonstrate that, if you don't pass it in, the component automatically goes with `default`.

Reply:
It's optional. There's a story right above that demonstrates that.
The `text` function here allows you to update the style dynamically in Storybook:
![Screen Shot 2019-04-03 at 3 10 40 PM](https://user-images.githubusercontent.com/1280564/55506183-c970cb00-5622-11e9-90c9-831cd7c248ce.png)